cudamapper/src/main.cu

```diff
@@ -363,6 +369,9 @@ void help(int32_t exit_code = 0)
               << R"(
         -F --filtering-parameter
             filter all representations for which sketch_elements_with_that_representation/total_sketch_elements >= filtering_parameter), filtering disabled if filtering_parameter == 1.0 [1'000'000'001] (Min = 0.0, Max = 1.0))"
+              << R"(
+        -o --output-dir
```

Comment:
This is confusing, the user would usually expect `-o --output-dir` to be the output location of all files not just intermediate ones. Why don't we output all files into this directory?